src/MaterialProperties/MaterialProperties.jl

```diff
@@ -89,6 +89,21 @@ material_properties[:CdZnTe] = (
     ϵ_r = 10.9,
     ρ = 5.78u"g*cm^-3"
 )
+
+abstract type CsPbBr3 <: AbstractDriftMaterial end
+Symbol(::Type{CsPbBr3}) = :CsPbBr3
+material_properties[:CsPbBr3] = (
+    name = "CsPbBr3",
+    E_ionisation = 6.9u"eV",
+    ϵ_r = 16.46,
+    ρ = 4.73u"g*cm^-3"
+)
+
+material_properties[:Pb] = (
+    name = "Lead",
+    ϵ_r = 1,

```

Comment:
Is this vale for lead correct? I would have expected a higher number

Reply:
As I know relative permitive of metals is equal to 1 for all metals. I know that for the you have an aluminum foil with 10.8 value, but for the clear metals is one as I [see](https://en.wikipedia.org/wiki/Relative_permittivity#:~:text=r%20remains%20unitless\).-,Metals,relative%20permittivity%20equal%20to%20one.) . Aluminum foil is not a clear aluminum, so probably that's why we see that.

Reply:
In general I add lead just as material for collimator. Electric field should not be applied to it.

Reply:
This way, „lead“ will be treated as if it was vacuum in the field calculation. This is fine if the detector is shielded from the collimator by some cryostat. But the presence of materials in the vicinity of the detector can have a significant impact on the field (also inside of the detector if it has a lot of uncontacted surface)

Reply:
I agree with you. It is better to put some huge value. I have done it, and have split request into two commits.